### ISSUE TITLE
Tomography appointment is happening at level 2 rather than level 3

### ISSUE THREAD

**Issue by BinglingICL** [UCL/TLOmodel#1499]
Scenario run results shows that Tomography appointment is happening at level 2 rather than at level 3. As defined by ResourceFile_Appt_Time_Table.csv, this appointment should happen and only happen at level 3/central hospitals.

Initial search shows that Tomography is only used by RTI module; but not yet know the cause of this inconsistent place of level.

**Comment by mnjowe** [comment]
Hi Bingling. I think [this code](https://github.com/UCL/TLOmodel/blob/3599cdc477cc195799bc6677753f25729c75664f/src/tlo/methods/rti.py#L3347-L3352) has to be changed to
```
if  'Tomography' in list(self.EXPECTED_APPT_FOOTPRINT.keys()):
            self.ACCEPTED_FACILITY_LEVEL = '3'
            self.add_equipment({'Computed Tomography (CT machine)', 'CT scanner accessories'})

self.add_equipment(self.healthcare_system.equipment.from_pkg_names('X-ray'))
```
I think with the current implementation, [this condition](https://github.com/UCL/TLOmodel/blob/3599cdc477cc195799bc6677753f25729c75664f/src/tlo/methods/rti.py#L3350) will never run since the first block will always be true i.e. `DiagRadio` will always be in the `list(self.EXPECTED_APPT_FOOTPRINT.keys())`. This is according to how the expected footprint is being initialised [here](https://github.com/UCL/TLOmodel/blob/3599cdc477cc195799bc6677753f25729c75664f/src/tlo/methods/rti.py#L3339) but also the fact [that this function](https://github.com/UCL/TLOmodel/blob/3599cdc477cc195799bc6677753f25729c75664f/src/tlo/methods/rti.py#L3345) is just adding another key/appointment `Tomography` to the existing footprint if there are any injuries requiring tomography appointment and overwriting `DiagRadio` if any injuries requiring its appointment

I think the above is the reason you're having tomography appointment happening at facility level 2. It is because [this block is not running](https://github.com/UCL/TLOmodel/blob/3599cdc477cc195799bc6677753f25729c75664f/src/tlo/methods/rti.py#L3351C12-L3352C15) hence no change of facility level

**Comment by BinglingICL** [comment]
Thanks very much @mnjowe for finding out the reason and providing the solution. Would raise a PR to fix it accordingly. Many thanks and sorry for not getting back on this earlier.

**Comment by BinglingICL** [comment]
Hi @tbhallett, just saw your label; would you like Mariana's team to raise this PR?

**Comment by tbhallett** [comment]
> Hi [@tbhallett](https://github.com/tbhallett), just saw your label; would you like Mariana's team to raise this PR?

Not necessarily. I just went through all the issues and applied labels as relevant. I think about 30 issues have the “good first issue” label now. I wasn’t sure if you had a fix in the works on this or not and hadn’t noticed your comment on this one earlier today. I wasn’t saying it was urgent to do that. I just thought the issues tracker could do with a “spring clean”!

**Comment by BinglingICL** [comment]
> > Hi [@tbhallett](https://github.com/tbhallett), just saw your label; would you like Mariana's team to raise this PR?
> 
> Not necessarily. I just went through all the issues and applied labels as relevant. I think about 30 issues have the “good first issue” label now. I wasn’t sure if you had a fix in the works on this or not and hadn’t noticed your comment on this one earlier today. I wasn’t saying it was urgent to do that. I just thought the issues tracker could do with a “spring clean”!

Thanks Tim. I have raised a PR for an "early spring clean" :) Mariana's team will have many other interesting issues to explore!
